src/index.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ** This file is automatically generated by gapic-generator-typescript. **
+// ** https://github.com/googleapis/gapic-generator-typescript **
+// ** All changes to this file may be overwritten. **
+
+import * as v1Module from './v1';
+import * as v1p1beta1Module from './v1p1beta1';
+import * as v1p2beta1Module from './v1p2beta1';
+import * as v1p3beta1Module from './v1p3beta1';
+import * as v1p4beta1Module from './v1p4beta1';
+import * as helpers from './helpers';
+// tslint:disable-next-line no-any
+const gapic: any = Object.freeze({
+  v1: v1Module,
+  v1p1beta1: v1p1beta1Module,
+  v1p2beta1: v1p2beta1Module,
+  v1p3beta1: v1p3beta1Module,
+  v1p4beta1: v1p4beta1Module,
+});
+
+// Augment the ImageAnnotatorClient objects with the helpers.
+for (const gapicVersion of Object.keys(gapic)) {
+  const clientProto = gapic[gapicVersion].ImageAnnotatorClient.prototype;
+  Object.assign(clientProto, helpers.call(gapicVersion));
+}
+const v1 = gapic.v1;
+const ImageAnnotatorClient = v1.ImageAnnotatorClient;
+const ProductSearchClient = v1.ProductSearchClient;
+const v1p1beta1 = gapic.v1p1beta1;
+const v1p2beta1 = gapic.v1p2beta1;
+const v1p3beta1 = gapic.v1p3beta1;
+const v1p4beta1 = gapic.v1p4beta1;
+export {
+  v1,
+  v1p1beta1,
+  v1p2beta1,
+  v1p3beta1,
+  v1p4beta1,
+  ImageAnnotatorClient,
+  ProductSearchClient,
+};
```

Comment:
should we also have `export default` for compatibility?

Reply:
sure, thank you!